README.adoc

```diff
@@ -259,11 +262,20 @@ information that is received from the Server-Sent Events.
 === Subscribing to SSE
 
 The [hotspot=initSSE file=0]`initSSE()` method is called when the page first
-loads. This subscribes the client to the SSE by creating a new instance of
-[hotspot=eventSource file=0]`EventSource` interface and specifying the `/bff/sse`
-endpoint in the parameter. The [hotspot=eventSource file=0]`EventSource` interface
-makes a `GET` request to this endpoint with a request header of `Accept:
-text/event-stream` to connect to the server.
+loads. This subscribes the client to the SSE by creating a new instance of the
```

Comment:
**change**
This subscribes the client to the SSE by creating a new instance of the...
**to**
This method subscribes the client to the SSE by creating a new instance of the